Fix/skeleton table row

jmhar3 opened the pull request.

Commit:
include uuid key

Commit:
install uuid

Commit:
format

lukerollans commented on src/components/display/SkeletonTableBody.tsx:
```diff
@@ -15,7 +16,7 @@ export const SkeletonTableBody = (props: SkeletonTableBodyProps) => {
   const skeletonTableRows = React.useMemo(() => {
     return Array.from(new Array(rowCount), () => (
-      <SkeletonTableRow columnCount={columnCount} />
+      <SkeletonTableRow columnCount={columnCount} key={uuid()} />
```
@jmhar3 instead of installing a package, I think this is an instance where passing an `index` to the key prop is OK. typically it's recommended not to do this as if the underlying array's sort/order changes our components end up with the wrong keys, but that isn't a concern here

the function passed as `Array.from`'s second argument is itself passed the current element's value, so we can do this...

```jsx
const skeletonTableRows = React.useMemo(() => {
  return Array.from(new Array(rowCount), (rowNumber) =>  ...)
}, [rowCount])
```

lukerollans reviewed (changes requested):
- [x] Remove `uuid` package
- [x] Pass current row number as the component's key prop

lukerollans committed:
Use row number as component key

Usually this is bad as it can cause bugs if the order of `index`
changes. Given that's very unlikely to happen here, the tradeoff
of uninstalling an external dep is worthwhile

lukerollans committed:
Remove uuid package

lukerollans committed:
Merge remote-tracking branch 'origin/canon' into fix/skeletonTableRow

lukerollans committed:
fix: reorder package.json attributes

lukerollans reviewed (approved):
@jmhar3 I took take of the changes quickly given you're snowed under!

lukerollans commented:
:tada: This PR is included in version 2.0.1 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/@perxhealth/react-ui/v/2.0.1)
- [GitHub release](https://github.com/perxhealth/react-ui/releases/tag/v2.0.1)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
